sources/install/package_ad.sh

```diff
@@ -1287,6 +1287,14 @@ function install_ntlm_theft() {
     add-to-list "ntlm_theft,https://github.com/Greenwolf/ntlm_theft,A tool for generating multiple types of NTLMv2 hash theft files"
 }
 
+function install_abuseACL() {
```

Comment:
It seems like you don't need any alias, please add the following line to whitelist this behavior:
`# CODE-CHECK-WHITELIST=add-aliases`
More info on the contributors doc: https://exegol.readthedocs.io/en/latest/community/contributors.html#adding-a-new-tool